src/status_im/utils/wallet_connect.cljs

```diff
@@ -0,0 +1,32 @@
+(ns status-im.utils.wallet-connect
+  (:require ["@walletconnect/client" :refer [CLIENT_EVENTS] :default WalletConnectClient]
+            ["@react-native-community/async-storage" :default AsyncStorage]
+            [clojure.string :as string]))
+
+(def default-relay-provider "wss://relay.walletconnect.org")
```

Comment:
Config is a better place for urls and metadata.

Reply:
Moved.